couchdb/templates/persistentvolumeclaim.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if and .Values.persistentVolume.enabled .Values.persistentVolume.existing -}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.persistentVolume.claimName }}
+  labels:
+    app: {{ template "couchdb.name" . }}
+    release: {{ .Release.Name }}
+  {{- with .Values.persistentVolume.annotations }}
+  annotations:
+    {{- toYaml . | nindent 10 }}
```

Comment:
nindent value here is probably 6 rather than 10? I assume it is 10 since it was copied from the statefulset yaml?

It feels like there's some potential for the persistentvolumeclaim.yaml and statefulset.yaml to get out of sync - perhaps we could extract a helper function for the common yaml?

Reply:
Good catch, thanks.  Moved these repeated sections to a couple helper functions.